header:
  seq: 0
  stamp:
    secs: 0
    nsecs:         0
  frame_id: /world
joint_names: ['robotiq_85_left_knuckle_joint']
points:
  -
    positions: [0.804]
    velocities: [0.0]
    accelerations: [0.0]
    effort: []
    time_from_start:
      secs: 0
      nsecs:         0
  -
    positions: [0.0]
    velocities: [0.0]
    accelerations: [0.0]
    effort: []
    time_from_start:
      secs: 2
      nsecs: 0
